type-definitions/most.d.ts

```diff
@@ -1,4 +1,5 @@
 declare type SeedValue<S, V> = { seed: S, value: V };
+declare type UnfoldValue<S, V> = SeedValue<S,V> | { done: boolean }
```

Comment:
Since the docs are kind of loose, we may need to be even more permissive.  Maybe we can go with this, and then relax it if it causes someone some pain.

Seems like we should tighten the done case to `{ done: true }` since it'd be pointless to return `{ done: false }`.  What do you think?